README.md

```diff
@@ -24,3 +33,96 @@ For the documentation of the module in `README.md`, the following guidelines app
 
 For the short module description (`shortdesc:`), the following guidelines apply:
 - The description should start with a phrase like "The glorb module reticulates splines" or "The tree module can be used to plant trees".
+
+### Post-install Module Config
```

Comment:
Should this be called "Local module configuration" instead, as the "local" term is already used?

Reply:
This section should also:
- Make the purpose of the configuration as storing data for the module to use when running post-boot clearer.
- Provide some tips and example pieces of code for the implementation.